testsuite/tests/ui/auth/conftest.py

```diff
@@ -74,8 +79,7 @@ def auth0_bounce_login(auth0_setup, navigator, browser, auth0_user):
 
 # pylint: disable=too-many-locals
 @pytest.fixture(scope="module")
-def rhsso_setup(request, custom_admin_login, rhsso_service_info, ui_sso_integration, navigator, testconfig,
-                threescale):
+def rhsso_setup(custom_admin_login, rhsso_service_info, ui_sso_integration, navigator, testconfig, threescale):
```

Comment:
this is almost unrelated change. The goal isn'ŧ to fix "everything", it is about not introducing more "antipaterns". In this particular case, rather than rewrite of finalization it's better to just add missing condition (if I understand the change properly). Keeping commit (and PR) as small as possible containing just relevant changes has highest priority